Dockerfile.template

```diff
@@ -1,3 +1,31 @@
+FROM golang:1.19-alpine{{ .alpine }} AS build
+
+RUN apk add --no-cache git make
+
+ENV NOTARYPKG github.com/theupdateframework/notary
```

Comment:
```suggestion
ENV NOTARYPKG github.com/notaryproject/notary
```

Reply:
This must match what is in the go manifest which in version 0.7.0 is still that one as per:

https://github.com/notaryproject/notary/blob/b0b6bfdd4933081e8d5ae026b24e8337311dd598/go.mod#L1

Reply:
They are the same repo, theupdateframework/notary redirects to notaryproject/notary, as your link shows.

Reply:
Right, but this is used in the Go modules, so it has to match what _Go_ thinks the module name is 🙃

Reply:
Exactly! It's confusing I know

Reply:
Careful; the module wasn't renamed yet in the current release though (still uses `github.com/theupdateframework/notary`); https://github.com/notaryproject/notary/blob/v0.7.0/go.mod#L1

i.e. `github.com/notaryproject/notary` did not yet do a release